### ISSUE TITLE
Duplicate meta elements in header on both client and production sites.

### ISSUE THREAD

**Issue by whazell** [ccswbs/hjckrrh#482]
The charset tag in the header is being duplicated, in both client and production sites is causing markup validation errors:

```
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
```

This seems to be being generated in: hjckrrh/profiles/ug/themes/ug/ug_cornerstone/templates/html.tpl.php, line 50-52.

```
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <?php print $head; ?>
```

This issue is causing failures for section 4.1.1 of the WCAG
